arcade/examples/gl/using_random_glsl.py

```diff
@@ -0,0 +1,100 @@
+from time import time
+from array import array
+from random import random
+
+from arcade import Window
+from arcade.gl import BufferDescription
+from arcade.resources import resolve_resource_path
+
+"""
```

Comment:
This docstring needs to be moved to the top of the file before imports for CI to pick it up.